### PR TITLE
Update collateral shortfall forecast new

### PR DESCRIPTION
**Pull request by arielibaba**

### CODE REVIEW

File: notebooks/collateral-shortfall-forecast/notebooks/0-download-stock-prices-data.ipynb [hunk 1]
```diff
@@ -5,67 +5,52 @@
    "id": "c7847127",
```
Comment (HuifangYeo) [inline notebooks/collateral-shortfall-forecast/notebooks/0-download-stock-prices-data.ipynb:5]: <p>multi indexed &gt; multi-indexed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (HuifangYeo) [inline notebooks/collateral-shortfall-forecast/notebooks/0-download-stock-prices-data.ipynb:5]: <p>multi index &gt; multi-index</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

File: notebooks/collateral-shortfall-forecast/notebooks/3-data-exploration-partial-autocorrelations.ipynb [hunk 2]
```diff
@@ -0,0 +1,467 @@
+{
```
Comment (HuifangYeo) [inline notebooks/collateral-shortfall-forecast/notebooks/3-data-exploration-partial-autocorrelations.ipynb:1]: <p>Do we need to bold the paragraph?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (HuifangYeo) [inline notebooks/collateral-shortfall-forecast/notebooks/3-data-exploration-partial-autocorrelations.ipynb:1]: <p>Can we keep the size of the fonts consistent for the paragraphs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (HuifangYeo) [inline notebooks/collateral-shortfall-forecast/notebooks/3-data-exploration-partial-autocorrelations.ipynb:1]: <p>diffrent &gt; different?</p><p>significatnt &gt; significant?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: notebooks/collateral-shortfall-forecast/notebooks/4-create-machine-learning-pipeline.ipynb [hunk 3]
```diff
@@ -0,0 +1,1536 @@
+{
```
Comment (HuifangYeo) [inline notebooks/collateral-shortfall-forecast/notebooks/4-create-machine-learning-pipeline.ipynb:1]: <p>Do we the paragraph to be bold?</p><p>- Roll the Time Series Datasets &gt; Roll the time series datasets</p><p>- Extract the "Relevant" Features &gt; Extract the "relevant" features</p><p>- Build and Train the Forecasting Models &gt; Build and train the forecasting models</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (HuifangYeo) [inline notebooks/collateral-shortfall-forecast/notebooks/4-create-machine-learning-pipeline.ipynb:1]: <p>Can you explain what does "Roll the Datasets" mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

File: notebooks/collateral-shortfall-forecast/notebooks/0-download-stock-prices-data.ipynb (hunk 1 above)
Comment (arielibaba) [reply notebooks/collateral-shortfall-forecast/notebooks/0-download-stock-prices-data.ipynb:5]: <p>Done</p>

Comment (arielibaba) [reply notebooks/collateral-shortfall-forecast/notebooks/0-download-stock-prices-data.ipynb:5]: <p>Done</p>

**Comment by arielibaba** [comment]
<p>Done</p>

---

 <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by arielibaba** [comment]
<p>Done</p>

---

 <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by arielibaba** [comment]
<p>Done</p>

---

 <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by arielibaba** [comment]
<p>Optional</p>

---

 <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by arielibaba** [comment]
<p>Done</p>

---

 <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by arielibaba** [comment]
<p>Corrected</p>

---

 <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by arielibaba** [comment]
<p>Done</p>

---

 <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: notebooks/collateral-shortfall-forecast/notebooks/3-data-exploration-partial-autocorrelations.ipynb (hunk 2 above)
Comment (arielibaba) [reply notebooks/collateral-shortfall-forecast/notebooks/3-data-exploration-partial-autocorrelations.ipynb:1]: <p>Optional</p>

Comment (arielibaba) [reply notebooks/collateral-shortfall-forecast/notebooks/3-data-exploration-partial-autocorrelations.ipynb:1]: <p>Optional</p>

Comment (arielibaba) [reply notebooks/collateral-shortfall-forecast/notebooks/3-data-exploration-partial-autocorrelations.ipynb:1]: <p>Done</p>

File: notebooks/collateral-shortfall-forecast/notebooks/4-create-machine-learning-pipeline.ipynb (hunk 3 above)
Comment (arielibaba) [reply notebooks/collateral-shortfall-forecast/notebooks/4-create-machine-learning-pipeline.ipynb:1]: <p>Optional</p>

[the remark this answers is not in the document]
Comment (arielibaba) [reply notebooks/collateral-shortfall-forecast/notebooks/4-create-machine-learning-pipeline.ipynb:1]: <p>Optional</p>

Comment (arielibaba) [reply notebooks/collateral-shortfall-forecast/notebooks/4-create-machine-learning-pipeline.ipynb:1]: <p>Added</p>

[the remark this answers is not in the document]
Comment (arielibaba) [reply notebooks/collateral-shortfall-forecast/notebooks/4-create-machine-learning-pipeline.ipynb:1]: <p>Done</p>

**Comment by arielibaba** [comment]
<p>Added</p>

---

 <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by arielibaba** [comment]
<p>yes</p>

---

 <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by arielibaba** [comment]
<p>Done</p>

---

 <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by arielibaba** [comment]
<p>Done</p>

---

 <a href='https://app.reviewnb.com/atoti/notebooks/pull/336/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Pull request closed without merging**